styles/bsky/catppuccin.user.css

```diff
@@ -0,0 +1,331 @@
+    }
+
+    /* fixes a contrast issue when the suggested follows button is enabled (very ugly selector) */
+    button[data-testid="suggestedFollowsBtn"][style*="background-color: rgb(255, 255, 255)"]
+      > svg
+      > path {
+      fill: @base !important;
+    }
+
+    /* text inside most buttons */
+    button > div > div[style~="color: rgb(0, 0, 0)"] {
+      color: @base !important;
+    }
+
+    /* search bar text */
+    [role="search"],
+    [data-testid="confirmCodeInput"] {
+      --placeholderTextColor: @subtext1 !important;
+    }
+
+    /* search bar icon */
+    svg[style^="color: rgb(141, 142, 150)"] {
+      color: @subtext1 !important;
+    }
+
+    /* small icons */
+    path[fill="#ffffff"] {
+      fill: @text;
+    }
+    path[fill="#fff"] {
+      fill: @text;
+    }
+
+    /* more gray small icons (seems to only be the trash can/delete icon) */
+    path[fill="#8D8E96"] {
+      fill: @subtext1;
+    }
+
+    /* settings button at the top of the timeline */
+    path[fill="#B9B9C1"] {
+      fill: @subtext1;
+    }
+
+    /* likes, reposts and comment counts under posts */
+    [style*="color: rgb(112, 116, 137)"] {
+      color: @subtext0 !important;
+    }
+
+    /* onboarding: continue/next button */
+    [data-testid="continueBtn"] {
+      background-color: @accent-color !important;
+    }
+    [data-testid="continueBtn"] > div > div {
+      color: @base !important;
+    }
+    [data-testid="continueBtn"] > div > svg > path {
+      fill: @base !important;
+    }
+
+    /* ... button under posts */
+    path[fill="#707489"] {
+      fill: @subtext0;
+    }
+
+    /* pretty self explanatory, user mentions in the post create modal */
+    .mention {
+      color: @accent-color;
+    }
+
+    /* mentions in the notification tab (highlighted notifications) */
+    div[style^="background-color: rgb(0, 16, 64)"] {
+      background-color: fadeout(@accent, 70%) !important;
+    }
+
+    /* generic blue background color inline (used for buttons in notifications tab) */
+    button[style*="background-color: rgb(0, 133, 255)"] {
+      background-color: @accent-color !important;
+    }
+    /* contrast fix for above */
+    button[style*="background-color: rgb(0, 133, 255)"] > div > div {
+      color: @base !important;
+    }
+
+    /* other button in notifications tab, inner text */
+    div[style*="color: rgb(82, 172, 254)"] {
+      color: @accent-color !important;
+    }
+
+    /* "No notifications yet!" text in the notifications tab */
+    div[style*="color: rgb(141, 142, 150)"] {
+      color: @subtext0 !important;
+    }
+  }
+
+  @media (prefers-color-scheme: light) {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+  @media (prefers-color-scheme: dark) {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+
+  /* manually set colorscheme */
+  .light-theme {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+  .dark-theme {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
```

Comment:
```suggestion
```

Unless these classes are actually used for dark/light mode you can delete this.

Reply:
supporting bsky's light mode would essentially mean rewriting half(?) of the userstyle, would it be fine if i just instruct people to set their bsky to dark mode even if they want to use latte? (and make the userstyle only have one theme toggle)

Reply:
(for context, latte seems to work perfectly fine with bsky set to dark mode)

Reply:
also @uncenter im able to provide you with an invite code if you need one (for testing the theme or general usage)

Reply:
That would be helpful, thank you.